CONTRIBUTORS.md

```diff
@@ -109,6 +111,7 @@ GitHub contributors:
 - BigZ
 - BluTiGeS
 - BMS-fenaille
+- boherm
```

Comment:
Ah you should have used Boris full name, but it can be handled in the next release along with the update of the mailmap no worries